spec.html

```diff
@@ -29918,68 +29930,68 @@ <h1>Runtime Semantics: GetSubstitution ( _matched_, _str_, _position_, _captures
-                  0x0024, 0x003C
+                  0x0024, N1, N2
+                  <br>
+                  where 0x0030 &le; N1 &le; 0x0039 and 0x0030 &le; N2 &le; 0x0039
                 </td>
                 <td>
-                  `$&lt;`
+                  `$`_nn_
+                  where _nn_ is a two-element expansion of |DecimalDigits|
                 </td>
                 <td>
-                  <emu-alg>
-                    1. If _namedCaptures_ is *undefined*, the replacement text is the String `"$&lt;"`.
-                    1. Else,
-                      1. Scan until the next `>` U+003E (GREATER-THAN SIGN).
-                      1. If none is found, the replacement text is the String `"$&lt;"`.
-                      1. Else,
-                        1. Let _groupName_ be the enclosed substring.
-                        1. Let _capture_ be ? Get(_namedCaptures_, _groupName_).
-                        1. If _capture_ is *undefined*, replace the text through `>` with the empty string.
-                        1. Otherwise, replace the text through `>` with ? ToString(_capture_).
-                  </emu-alg>
+                  If _nn_ is `00` or the MV of _nn_ &gt; _m_, the replacement is the string-concatenation of the replacement for the first two matched code units (i.e., a replacement specified by one of the two preceding rows) and the remaining matched code units (i.e., a single |DecimalDigit|).
+                  <br>
+                  Otherwise, if the element in _captures_ at index equal to the MV of _nn_ minus 1 is *undefined*, the replacement is the empty String.
+                  <br>
+                  Otherwise, the replacement is the element in _captures_ at index equal to the MV of _nn_ minus 1.
                 </td>
               </tr>
               <tr>
                 <td>
-                  0x0024
+                  0x0024, 0x003C, any sequence of code units not containing 0x003E, 0x003E
                 </td>
                 <td>
-                  `$` in any context that does not match any of the above.
+                  `$&lt;` _groupName_ `>`
+                  where _groupName_ is a sequence of code units that does not contain a `>` U+003E (GREATER-THAN SIGN)
                 </td>
                 <td>
-                  `$`
+                  <emu-alg>
+                    1. Set _groupName_ to the String value whose code units are the code units of _groupName_.
+                    1. If _namedCaptures_ is *undefined*, then
+                      1. The replacement is the string-concatenation of the String `"$&lt;"`, ! GetSubstitution(_matched_, _str_, _position_, _captures_, _namedCaptures_, _groupName_), and the String `">"`.
+                    1. Else,
+                      1. Let _capture_ be ? Get(_namedCaptures_, _groupName_).
+                      1. If _capture_ is *undefined*, the replacement is the empty String.  Otherwise, the replacement is ? ToString(_capture_).
```

Comment:
```suggestion
                      1. If _capture_ is *undefined*, the replacement is the empty String. Otherwise, the replacement is ? ToString(_capture_).
```